packages/kit/types/index.d.ts

```diff
@@ -24,6 +24,8 @@ import type { PluginOptions } from '@sveltejs/vite-plugin-svelte';
 
 export { PrerenderOption } from './private.js';
 
+export const VERSION: `${number}.${number}.${number}`;
```

Comment:
```suggestion
/**
 * The current version of SvelteKit
 */
export const VERSION: `${number}.${number}.${number}`;
```

Reply:
if we ever use prerelease mode again, version could also be `2.0.0-next.1` etc